### DISCUSSION TITLE
How to preview the whole notebook in VSCode?

### DISCUSSION THREAD

**Discussion by Photonico** [microsoft/vscode-jupyter#9110] [Questions and Answers] [answered]
Hi,

I want to make a speech with the presentation in Jupyter format.
Demonstrating the dynamic file is untidy, exporting to HTML format is also untidy and tedious.
How to demonstrate a Jupyter notebook in Preview or Read-only table, like the Markdown extension preview .md files?

Sincerely,
Lu

---- Update ----

I found that the combination key `Ctrl` + `Shift` + `v` can invoke the preview of the cell.

**Comment by greazer** [comment]
Hi @Photonico, Unfortunately, I'm not entirely sure I'm following your question. I understand that you're not satisfied with how notebooks are presented while you are editing them and running them in VS Code. You would like to be able to edit them and see some sort of separate view of the same notebook that's a more cleaned up and tidy.

What I'm not understanding is what is it about the notebook you're editing that you feel is messy or untidy and shouldn't be presented in a separate view? What code cells or output shouldn't be shown in the preview? How would we know?

Are you possibly referring to the use of the Interactive Window with a Python file? The interactive window displays a history of what was executed and so it doesn't completely represent what is in the Python file.

<img width="793" alt="image" src="https://user-images.githubusercontent.com/11685408/155584409-a7a1bf9b-958c-47cd-bb1f-4b08b327567d.png">

**Reply by Photonico** [reply]
Hi,
 thanks for your reply. 
I use jupyter notebook file,  not the interactive window. 
[![](https://raw.githubusercontent.com/Photonico/Problems/master/2022/Screenshot_20220224_000000.png)]

**Comment by IanMatthewHuff** [comment, marked as answer]
@Photonico Are you looking for something like the RISE extension possibly? https://github.com/damianavila/RISE. We don't have anything like that in VS Code right now, but it's an interesting idea that we could consider adding.

**Reply by Photonico** [reply]
Thanks for your reply!
That's a good idea! I will try it.
